Makefile

```diff
@@ -0,0 +1,78 @@
+ifndef VERBOSE
+.SILENT:
+endif
+
+.DEFAULT_GOAL := help
+
+log_error = (>&2 echo "\x1B[31m>> Error:$1\x1B[39m\n" && $(MAKE) help && exit 1)
+
+.PHONY: help
+help:
+	@echo "Usage: make [os] [version] [command]"
+	@echo "\n"
+	@echo "Examples:"
+	@echo "  make ubuntu 2004 up"
+	@echo "  make windows server-2019 destroy"
+	@echo "  make windows server-2019 rdp"
+	@echo "  make ubuntu 2004 ssh"
+	@echo "\n"
+
+
+# Operating System Config
+
+# ============= Windows ============= #
+.PHONY: windows
+windows:
+	$(eval PLAT=windows)
+ifneq (,$(findstring up,$(MAKECMDGOALS)))
+	bolt module install --project ./windows/bootstrap
+endif
+
+server-2012R2:
+	$(eval VERSION=server-2012R2)
+
+server-2016:
+	$(eval VERSION=server-2016)
+
+server-2019:
```

Comment:
Unrelated to this PR but `server-2019` now uses bolt provisioning which seems to confuse vagrant

`vm:
* The 'bolt' provisioner could not be found.

Vagrant:
* Unknown configuration section 'bolt'.`

The vagrant [docs](https://developer.hashicorp.com/vagrant/docs/provisioning/puppet_apply) don't have bolt listed as a known provisioner. Only `puppet apply` and `puppet agent`. 

Is this bolt provisioner a custom provisioner of some sort?

Reply:
Yeah you need to install the bolt plugin for vagrant.. i thought i'd updated the readme

Reply:
oh yes I did - its on the main readme under requirements.